client/components/deposits-information/index.tsx

```diff
@@ -106,31 +106,41 @@ const DepositsInformationOverview: React.FunctionComponent< OverviewProps > = (
 					</p>
 				</div>
 			</CardHeader>
-
-			<Flex className="wcpay-deposits-information-row" align="normal">
-				<DepositsInformationBlock
-					title={ __( 'Pending balance', 'woocommerce-payments' ) }
-					value={ formatCurrency( pendingAmount, currency ) }
-					children={ pendingDepositsLink }
-				/>
-				<DepositsInformationBlock
-					title={ __( 'Next deposit', 'woocommerce-payments' ) }
-					value={ formatCurrency( nextScheduledAmount, currency ) }
-					children={ nextScheduledLink }
-				/>
-			</Flex>
-			<Flex className="wcpay-deposits-information-row" align="normal">
-				<DepositsInformationBlock
-					title={ __( 'Last deposit', 'woocommerce-payments' ) }
-					value={ formatCurrency( lastPaidAmount, currency ) }
-					children={ lastPaidLink }
-				/>
-				<DepositsInformationBlock
-					title={ __( 'Available balance', 'woocommerce-payments' ) }
-					value={ formatCurrency( availableAmount, currency ) }
-					children={ depositButton }
-				/>
-			</Flex>
+			<CardBody>
```

Comment:
Using `CardBody` here for consistency. It didn't require any styling changes, but there may be redundant padding in styles for this component - reviewers please keep in mind!

Reply:
Side note: the name `DepositsInformationOverview` doesn't make sense to me, I think of this as the `Balances` card or maybe `CurrencySwitcher` card. 

Should we rename the main component? 

How would we refer to this UI in the documentation for merchants?

Reply:
This `DepositsInformationOverview` is the old UI component. The one that looks like this: 

<img width="699" alt="Screenshot 2023-04-12 at 11 17 38 am" src="https://user-images.githubusercontent.com/8490476/231322384-670f457b-5892-4d82-9ffd-94e411f71005.png">

Reply:
These changes are now gone – thanks for the tip @james-allan! I reviewed `AccountBalances` components and it's looking good (no need to use `CardBody`, `CardDivider` etc).

Reply:
Yep I was going to say the same. That card doesn't have any padding so we're all good there. :)